kokoro/presubmit.sh

```diff
@@ -17,4 +17,4 @@ docker kill $(docker ps --all --quiet) || true
 cd github/jib
 
 # we only run integration tests on jib-core for presubmit
-./gradlew clean build :jib-core:integrationTest --info --stacktrace
+./gradlew clean build :jib-gradle-plugin:integrationTest --info --stacktrace
```

Comment:
Adding temporary check to verify the change in the CI. This needs to be removed before merging.